Jenkinsfile

```diff
@@ -36,19 +37,16 @@ for(i = 0; i < buildTypes.size(); i++) {
                 // Now run the actual build.
                 stage("${buildType} Build / Test") {
                     timeout(time: 180, unit: 'MINUTES') {
-                        // See below for what this method does - we're passing an arbitrary environment
-                        // variable to it so that JAVA_OPTS and MAVEN_OPTS are set correctly.
-                        withMavenEnv(["JAVA_OPTS=-Xmx1536m -Xms512m",
-                                    "MAVEN_OPTS=-Xmx1536m -Xms512m"]) {
-                            // Actually run Maven!
-                            // -Dmaven.repo.local=… tells Maven to create a subdir in the temporary directory for the local Maven repository
-                            def mvnCmd = "mvn -Pdebug -U -Dset.changelist help:evaluate -Dexpression=changelist -Doutput=$changelistF clean install ${runTests ? '-Dmaven.test.failure.ignore' : '-DskipTests'} -V -B -Dmaven.repo.local=$m2repo -s settings-azure.xml -e"
-                            if(isUnix()) {
-                                sh mvnCmd
-                                sh 'test `git status --short | tee /dev/stderr | wc --bytes` -eq 0'
-                            } else {
-                                bat mvnCmd
-                            }
+                        // We're passing an arbitrary environment variable to it so that JAVA_OPTS and MAVEN_OPTS are set correctly.
+                        infra.runMaven(["-Pdebug", "-U", "-Dset.changelist", "help:evaluate",
+                                        "-Dexpression=changelist", "-Doutput=$changelistF",
+                                        runTests ? '-Dmaven.test.failure.ignore' : '-DskipTests',
+                                        "-V", "-B", "-Dmaven.repo.local=$m2repo",
```

Comment:
`runMaven` includes `-B`

Reply:
```suggestion
                                        "-V", "-Dmaven.repo.local=$m2repo",
```

Reply:
Did you mean to suggest a change to your own PR?